core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -1,147 +1,171 @@
 <div class="splash-page">
-  <div class="oppia-splash-section-one" dir="auto">
-    <div class="oppia-splash-section-one-left" tabindex="0">
-      <h1 class="oppia-splash-h1" [innerHTML]="'I18N_SPLASH_TITLE' | translate"></h1>
-      <div class="oppia-splash-h2-container">
-        <h2 class="oppia-splash-h2 oppia-splash-h2-text" [innerHTML]="'I18N_SPLASH_SUBTITLE' | translate"></h2>
-      </div>
-      <div class="oppia-splash-button-container">
-        <button type="submit" (click)="onClickBrowseLessonsButton()"
-                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200">
-          {{ 'I18N_ACTION_BROWSE_LESSONS' | translate }}
-        </button>
-      </div>
-      <div class="oppia-splash-button-container">
-        <button type="submit" (click)="onClickAccessAndroidButton()"
-                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200">
-          {{ 'I18N_ACTION_ACCESS_ANDROID_APP' | translate }}
-        </button>
-      </div>
-    </div>
-    <div class="oppia-splash-section-one-right">
-      <picture class="oppia-splash-main-image" *ngIf="!isWindowNarrow">
-        <source type="image/webp" [srcset]="getImageSet('/splash/splashMainDesktop', 'webp')">
-        <source type="image/png" [srcset]="getImageSet('/splash/splashMainDesktop', 'png')">
-        <img [src]="getStaticImageUrl('/splash/splashMainDesktop1x.png')"
-             width="800"
-             height="663"
-             alt="Photo of two kids and a tutor">
-      </picture>
-      <picture class="oppia-splash-main-image" *ngIf="isWindowNarrow">
-        <source type="image/webp" [srcset]="getStaticImageUrl('/splash/splashMainMobile.webp')">
-        <source type="image/png" [srcset]="getStaticImageUrl('/splash/splashMainMobile.png')">
-        <img [src]="getStaticImageUrl('/splash/splashMainMobile.png')"
-             width="629"
-             height="415"
-             alt="Photo of a girl in front of computer">
-      </picture>
-    </div>
-  </div>
-
-  <div class="oppia-splash-section-two">
-    <div class="oppia-splash-icon-1-container">
-      <img [src]="getStaticImageUrl('/splash/userIcon.svg')"
-           class="oppia-splash-icon-1"
-           width="35"
-           height="35"
-           alt="">
-      <div class="splash-icon-text-container">
-        <span class="splash-icon-text"
-              [innerHTML]="'I18N_SPLASH_ICON_ONE_TEXT' | translate">
-        </span>
-      </div>
+  <div class="oppia-splash-features-section" dir="auto" role="region" >
```

Comment:
Hi @AkashPaloju -- would it be possible to make a splash-page-one.component.html and a splash-page-two.component.html, then use a feature flag value to ng-if between those? That way we can get some validation of this page before we launch it fully, so that we can make sure it is better than the current one.

We can keep that running for a couple months and then analyze the results at the end to decide which version to go with finally. OK for both these pages to use the same JS component if that makes things easier, and OK for these pages to contain duplicate code if needed since we're going to drop one anyway.